packages/documentation/docs/controls/charts/overview.md

```diff
@@ -43,9 +76,28 @@ noMargin
 
 ## Empty state
 
+An empty state occurs when a user first opens an application, and no chart was added manually. Make sure to provide a clear message and a call to action to guide users on what to do next.
+
 <Playground
 height="40rem"
 name="echarts-empty-state"
 noMargin
 examplesByName>
 </Playground>
+
+### Loading indicator
+
+A loading indicator provides users with visual feedback that the chart is being processed and will be displayed shortly. This helps manage user expectations and enhances the overall user experience by preventing confusion or frustration during data loading times. 
```

Comment:
Could you please check that again? Generally, you got a point. Maybe you could focus a bit more on the relevant, or unique parts, and instead kick out the chart-unspecific parts? (e.g. "manage user expectations" might not bring a lot of value to the guide)